docker-swarm-cluster/README.md

```diff
@@ -4,7 +4,8 @@
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 
-*(Built by: [ahmetalpbalkan](https://github.com/ahmetalpbalkan), [garimakhulbe](https://github.com/garimakhulbe))*
+*(Built by: [ahmetalpbalkan](https://github.com/ahmetalpbalkan),
```

Comment:
@ahmetalpbalkan Thanks! I looked through the git history and it seems the garimakhulbe wrote the other template https://github.com/Azure/azure-quickstart-templates/tree/master/docker-swarm-cluster-simple. Not sure how her name got in this readme.